### ISSUE TITLE
how to improve the quality of male's voice?

### ISSUE THREAD

**Issue by wotulong** [NVIDIA/tacotron2#234]
Thanks for your job.I have trained your code on my own dataset, as a result .I can get a good quality of voice when I train with female's voice, but get a poor quality of voice when using male's voice to train( both source voice have a very good quality, male 's data is easy to alignment when set tacotron_output_perstep=1, but male's data is hard to alignment). by the way ,I keep the hparams same as yours. Do you have any suggestions for me? Thank you.

**Comment by akshitmittal1** [comment]
what are issues are there with the male audio quality? When listening output audio, can we understand the words or we can't understand the words at all?

**Comment by wotulong** [comment]
> what are issues are there with the male audio quality? When listening output audio, can we understand the words or we can't understand the words at all?

Yes, we can understand every word in the voice.The male's voice have more reverb than female's.

**Comment by wotulong** [comment]
> > what are issues are there with the male audio quality? When listening output audio, can we understand the words or we can't understand the words at all?
> 
> Yes, we can understand every word in the voice.The male's voice have more reverb than female's.

and not as cleary as female's. sound like have some noise.

**Comment by akshitmittal1** [comment]
Okay, try to infer it at a larger sampling rate. That thing worked for me, it increases the pitch of the voice and makes it clearer.

**Comment by wotulong** [comment]
> Okay, try to infer it at a larger sampling rate. That thing worked for me, it increases the pitch of the voice and makes it clearer.

Thanks.My samplerate is 22050, what's yours?

**Comment by akshitmittal1** [comment]
It all depends on your training data as well as the hyperparameters. Did increasing sampling rate while infering work?

**Comment by rafaelvalle** [comment]
Please note that increasing the sampling rate should not alter the pitch of the voice nor necessarily make it cleaner. Increasing the sampling rate makes it possible to represent higher frequencies. 
One can represent frequencies up to half the sampling rate. For eample, in 16khz one can represent frequencies up to 8khz.

**Comment by rafaelvalle** [comment]
Closing due to inactivity.

**Comment by Approximetal** [comment]
I also have this problem as above said "The result get a poor quality of voice with reverb and sound like have some noise." It has trained 450k, and loss decreases slower now, should I continue training ? What can I adjust? Any other mothods could I try to improve the qulity? @rafaelvalle 
![image](https://user-images.githubusercontent.com/25227045/75073040-a1889f80-5533-11ea-9ab3-b859794207de.png)
![image](https://user-images.githubusercontent.com/25227045/75073121-c9780300-5533-11ea-9dd3-4432583563c4.png)

**Comment by Liujingxiu23** [comment]
I have the same problem as you @wotulong , male voice worse than female,  noise if male waves synthesized is perceptible, while these is no noise of female wavs. Have you find any solution or any reason?

**Comment by wotulong** [comment]
> I have the same problem as you @wotulong , male voice worse than female, noise if male waves synthesized is perceptible, while these is no noise of female wavs. Have you find any solution or any reason?

I think maybe it  because the voice of male has more energy in low freq, you can try decrease the value of "fmax" in hparams file.

**Comment by ksaidin** [comment]
Check [this issue](https://github.com/NVIDIA/tacotron2/issues/297) and it's solution. It is suggested that training waveglow with your own dataset produces original like output.

**Comment by tugstugi** [comment]
@wotulong which "fmax" have you tried for your male voice?

**Comment by wotulong** [comment]
> @wotulong which "fmax" have you tried for your male voice?

fmin=60, fmax=7000.
